src/store/index.js

```diff
@@ -1,18 +1,32 @@
-import { createStore, combineReducers } from 'redux'
+import { createStore, combineReducers, applyMiddleware } from 'redux'
+import { composeWithDevTools } from 'redux-devtools-extension';
 
 import products from './products/reducer'
 import cart from './cart/reducer'
+import customer from './customer/reducer'
+import { logger } from './middlewares/loggingMiddleware'
 
 const reducer = combineReducers({
   products,
   cart,
+  customer,
 })
 
+const initialState = {}
+const enhancers = []
+const middleware = [
+  logger,
+]
+
+const composedEnhancers = composeWithDevTools(
+  applyMiddleware(...middleware),
+  ...enhancers
+)
+
 const store = createStore(
   reducer,
-  // this variable will be set if you have redux-dev-tools extension installed in your browser
-  // https://github.com/zalmoxisus/redux-devtools-extension#11-basic-store
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
I believe there is no need to remove it. Actually, it's a very handy extension.

Reply:
I have not found a way how to combine it with other middleware. I though composeWithDevTool method replaces this.